chaosinstana/api.py

```diff
@@ -1,15 +1,44 @@
 # -*- coding: utf-8 -*-
+from typing import Dict
+
 from chaoslib.exceptions import ActivityFailed
 from chaoslib.types import Configuration, Secrets
 from logzero import logger
 import requests
 
+from chaosinstana.types import InstanaResponse
+
 
 __all__ = ['get_all_events', 'get_event']
 
 
+def get_request(url: str, params: Dict, secrets: Secrets) -> InstanaResponse:
```

Comment:
Prefer maybe "execute_instana_request" maybe?

Reply:
renamed to execute_instana_get_request